src/__tests__/Snapshot.test.tsx

```diff
@@ -0,0 +1,14 @@
+import '@testing-library/jest-dom/extend-expect';
+// import renderer from 'react-test-renderer';
+// import Active from '../pages/active';
+
+// checking test feature
+
+test('render correctly', () => {
+  expect('true').toBe('true');
+});
+
+// test('render correctly', () => {
+//   const tree = renderer.create(<Active />).toJSON();
+//   expect(tree).toMatchSnapshot();
+// });
```

Comment:
Can we write an actual Snapshot test for some page/component?
Can we not commit unused comments?